src/test/java/mate/academy/springbootintro/service/CategoryServiceTest.java

```diff
@@ -82,8 +82,8 @@ public void findAll_ValidPageable_ReturnsAllCategories() {
         // Given
         Category category = new Category();
         category.setId(VALID_CATEGORY_ID);
-        category.setName("Horror");
-        category.setDescription("Scary");
+        category.setName(CATEGORY_NAME);
+        category.setDescription(CATEGORY_DESCRIPTION);

```

Comment:
You should create and use something similar to createBookDto you created in BookControllerTest